App.xaml.cs

```diff
@@ -10,13 +11,32 @@ public partial class App : Application
     {
         private void Application_Startup(object sender, StartupEventArgs e)
         {
+            string[] availableLocale = { "en-US", "ja-JP" };
```

Comment:
Can we pull this from a config file or something? I believe we will also have to support Korean sometime. If that's the case, better to pull this from a config file.

Reply:
@VikasShashidhar I didn't find any better solution other than this.